src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1524,6 +1524,41 @@ internal static bool TryGetSpecialTypeMember<TSymbol>(CSharpCompilation compilat
             return true;
         }
 
+#nullable enable
+
+        /// <summary>
+        /// This is a layer on top of the Compilation version that generates a diagnostic if the well known
+        /// member isn't found.
+        /// </summary>
+        internal Symbol? GetWellKnownTypeMember(WellKnownMember member, BindingDiagnosticBag diagnostics, SyntaxNode syntax)
+        {
+            return TryGetWellKnownTypeMember<Symbol>(this.Compilation, member, syntax, diagnostics, out var memberSymbol)
+                ? memberSymbol
+                : null;
+        }
+
+        internal static bool TryGetWellKnownTypeMember<TSymbol>(CSharpCompilation compilation, WellKnownMember wellKnownMember, SyntaxNode syntax, BindingDiagnosticBag diagnostics, [MaybeNullWhen(false)] out TSymbol symbol)
```

Comment:
>TryGetWellKnownTypeMember [](http://example.com/codeflow?start=29&length=25)

Could this method be replaced with one of the existing `GetWellKnownTypeMember()` overloads, for instance the overload at line 1664?